internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -478,9 +478,7 @@ func checkIntValueRange(valueType models.ValueType, val int64) bool {
 			isValid = true
 		}
 	case models.Int64:
-		if val >= math.MinInt64 && val <= math.MaxInt64 {
-			isValid = true
-		}
+		isValid = true
```

Comment:
The code of checkIntValueRange function has been copied from edgex's [device-rest-go](https://github.com/edgexfoundry/device-rest-go/blob/cbc1fb126fca16d22f986b5cd5c305caa6710614/driver/resthandler.go#L324).
What about contributing this code to Edgex for easy maintenance?

Reply:
see above